### ISSUE TITLE
Add support for disabling top-scroll for partials

### ISSUE THREAD

**Issue by Eason0729** [denoland/fresh#2781]
> I have seen this issue: https://github.com/denoland/fresh/issues/1923
> 
> However, that issue seems to be about `<form/>` support.

I am working on `load more` button with `<Partial mode="append">`, more entries are loaded but client view keep scrolling to top.

Maybe I should just reopen that issue?

**Comment by murphyslaw** [comment]
In my case, I wanted to stay on the same page andI was able to transform the link into a button without a form. Because of using the `button` element, it does not scroll to the top.

```javascript
// before
<a href="/current" f-partial="/partials/content">Link</a>
```

```javascript
// after
<button f-partial="/partials/content">Link</button>
```

**Comment by marvinhagemeister** [comment]
Yeah going with buttons is the right solution. Using anchor elements implies a navigation intend. Navigating to a new page should always scroll to top.
